### DISCUSSION TITLE
Flames Version 4.4.3 "Mild Face Implosion"

### DISCUSSION THREAD

**Discussion by SeveralCircles** [colorfulFlames/flames#66] [Announcements]
## What's Changed
* Update to API 1.1 by @SeveralCircles in https://github.com/colorfulFlames/flames/pull/61
* Fix #62 by @SeveralCircles in https://github.com/colorfulFlames/flames/pull/64

**Full Changelog**: https://github.com/colorfulFlames/flames/compare/v4.4.2...v4.4.3

<hr /><em>This discussion was created from the release <a href='https://github.com/colorfulFlames/flames/releases/tag/v4.4.3'>Flames Version 4.4.3 "Mild Face Implosion"</a>.</em>

**Comment by Minotaurinmaze** [comment]
Cring
![image](https://user-images.githubusercontent.com/65237985/137955297-2d7e9002-5ab1-4019-a367-0037e554e862.png)
